src/Standards/Generic/Sniffs/Operators/StrictComparisonSniff.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Ensures that constant names are all uppercase.
+ *
+ * @author    Vincent Langlet <vincentlanglet@exemple.com>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\Generic\Sniffs\Operators;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class StrictComparisonSniff implements Sniff
+{
+
+    /**
+     * Types to replace: key is operator to replace, value is operator to replace with.
+     *
+     * @var array
+     */
+    public $operators = [
+        T_IS_EQUAL     => '===',
+        T_IS_NOT_EQUAL => '!==',
+    ];
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return [
+            T_IS_EQUAL,
```

Comment:
What's the point of having the public property if you don't use it in `register()` ? People could now add additional operators via the public property, but they would never be sniffed for.

Reply:
Whoah, I'm tired. Thanks.